src/main/java/com/example/spring_bank_project/auth/infrastructure/config/AuthDomainWebSecurityConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.example.spring_bank_project.auth.infrastructure.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@Order(value = 101)
```

Comment:
Ordered.HIGHEST_PRECEDENCE / Ordered.LOWEST_PRECEDENCE mozna uzyc, ale nie widze powodu uzycia tutaj w ogole

Reply:
Musiałem użyć @Order ponieważ podczas budowania apki, gryzło się to z WebSecurityConfigiem z innej domeny

Reply:
Najprawdopodbniej definiowanie domentowych web configów to buła

Reply:
W następnym prku zamierzam ujednolicić to do jednego pliku